PennMobile/About/Controllers/AboutViewController.swift

```diff
@@ -22,6 +23,21 @@ class Member {
         }
         self.imageURL = URL(string: "https://s3.us-east-2.amazonaws.com/penn.mobile/about/" + image)
     }
+
+    static func < (lhs: Member, rhs: Member) -> Bool {
+        ///sort by last name then first
+        if let lhsLast = lhs.name.components(separatedBy: " ").last,
+            let rhsLast = rhs.name.components(separatedBy: " ").last {
+            if lhsLast != rhsLast {
+                return lhsLast < rhsLast
+            }
+        }
+        return lhs.name < rhs.name
+    }
+
+    static func == (lhs: Member, rhs: Member) -> Bool {
+        return lhs.name == rhs.name && lhs.imageURL == rhs.imageURL
+    }
```

Comment:
This isn't a huge issue, but my name, for example, has two spaces so writing a function like this is not the desired behavior (ie. my last variable will be "Min Choi" not "Choi")

Rather than splitting the name component, it might be better to replace the name variable with two variables: firstName and lastName. I think this would increase readability as you no longer have to include the components(seperatedBy: ) in the comparison.

Reply:
Good point!